components/budget/filters/PeriodFilter.js

```diff
@@ -154,8 +154,52 @@ const PeriodFilter = ({ onChange, value, inputId, minDate, ...props }) => {
               interval={tmpDateInterval}
             />
           </Box>
+          <StyledInputField
+            name="timezoneType"
+            mt="12px"
+            labelFontSize="13px"
+            labelFontWeight="700"
+            label={
+              <Flex alignItems="center">
+                <Span mr={1}>
+                  <FormattedMessage defaultMessage="Timezone" />
+                </Span>
+                <StyledTooltip
+                  content={
+                    <FormattedMessage defaultMessage="By default, all dates are filtered and displayed using your local timezone. You can switch to UTC to indicate that the dates provided above use the Coordinated Universal Time format, which matches how email reports are generated" />
+                  }
+                >
+                  <InfoCircle size={16} />
+                </StyledTooltip>
+              </Flex>
+            }
+          >
+            {inputProps => (
+              <StyledButtonSet
+                {...inputProps}
+                size="tiny"
+                items={['local', 'UTC']}
+                buttonProps={{ p: 1, fontSize: '11px' }}
```

Comment:
According to design,

```suggestion
                buttonProps={{ p: 1, fontSize: '13px', fontWeight: 400 }}
```